src/components/navbar.tsx

```diff
@@ -25,6 +26,9 @@ export default ({ platforms }: Props): JSX.Element => {
               API
             </SmartLink>
           </Nav.Item>
+          <Nav.Item>
+            <Nav.Link className="text-primary" href={getSandboxURL().toString()}>Demo</Nav.Link>
```

Comment:
@Neo-Zhixing why can't we use the same component as in the other PR?

Reply:
Because that one always returns `<a>` so it's easier to use in mdx files.